src/mumble/ConnectedQuitBehaviour.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021-2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_CONNECTEDQUITBEHAVIOUR_H
+#define MUMBLE_CONNECTEDQUITBEHAVIOUR_H
+
+#include <QtCore/QObject>
+#include <vector>
+
+
+/// This enum lists a series of quit options when connected to a server
+enum class ConnectedQuitBehaviourID {
+	ASK      = 0,
+	MINIMIZE = 1,
+	CLOSE    = 2
+};
+
+struct ConnectedQuitBehaviour {
+	ConnectedQuitBehaviourID id;
+	QString description;
+	QString explanation;
```

Comment:
```suggestion
	QString name;
	QString description;
```
I think that would be more fitting here.